src/resolve-masa.test.ts

```diff
@@ -0,0 +1,64 @@
+import { ResolveMasa } from './resolve-masa'
+import { Masa } from '@masa-finance/masa-sdk'
+import { providers } from 'ethers'
+
+const providerUrl = 'https://alfajores-forno.celo-testnet.org'
+
+describe('masa', () => {
+  describe('ResolveMasa', () => {
+    const resolver = new ResolveMasa({
```

Comment:
I might be missing something, but it looks like this does actual resolutions against names on alfajores.  Is that right?  If so, can you pass in some type of mock `Masa` instance instead that implements the behavior you're trying to test in `ResolveMasa`? Example from nomspace tests here: https://github.com/valora-inc/resolve-kit/blob/main/src/resolve-nom.test.ts#L6

We're trying to mock blockchain or network dependencies for unit tests in this repo.